src/Common/tests/Tests/System/StringTests.cs

```diff
@@ -3342,4 +3590,26 @@ public static unsafe void GetEnumeratorTest()
             Assert.False(chEnum.MoveNext(), "expect to not having any characters to enumerate");
         }
     }
+
+    public static class TestHelpers
```

Comment:
Can we put this class into a separate file?

Reply:
>Can we put this class into a separate file?

in Common tests folder?

Reply:
Yes. Unless it starts to conflict with the System.Memory TestHelpers. In which case, maybe rename it to CommonTestHelpers (here: https://github.com/dotnet/corefx/tree/master/src/Common/tests/System)